src/libraries/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -6,6 +6,7 @@
   <ItemGroup>
     <Compile Include="System\Collections\HashtableExtensions.cs" />
     <Compile Include="System\Collections\Generic\ValueListBuilder.Pop.cs" />
+    <Compile Include="System\Text\RegularExpressions\Symbolic\SymbolicRegexThresholds.cs" />
```

Comment:
NIT: Can we move this line down to where we  have all of the NonBacktracing files?

Reply:
Will do.

Reply:
Moved it right after 
`<Compile Include="System\Text\RegularExpressions\Symbolic\SymbolicRegexRunnerFactory.cs" />`
to keep it (roughly) in alphabetical order, reordered some lines a bit there.